CDCgov/prime-reportstream
# React spinner updates and Jest dependency fix

This PR updates our Spinner styling and placement, **and fixes a Jest dependency issue!**

Test Steps:
For Jest fix:
1. Run `npm start`.
2. If you receive a message about Jest 26.x.x being a dependency of `react-scripts`, I have failed. If not, you're welcome.

For spinners:
1. Login
2. Navigate to Daily Data
3. You should see a spinner while the table data loads and "Loading Info..." where the org name is until they're loaded.

## Changes
- Removed Jest 27.2.X `devDependency` from `package.json` since 26.x.x is a peer dependency of react-scripts already
- Removed `<Suspense>` from `App.tsx`
- Added `<Spinner>` component
- Used `<Suspense>` on `OrgName` and `TableData` in `Daily.tsx` to show loading indicators

## Checklist

### Testing

### Process

## Fixes
- #2044

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
No
- Does this PR include changes in authentication and/or authorization of existing endpoints?
No
- Does this change introduce new dependencies that need vetting?
No
- Does this change require changes to our infrastructure?
No
- Does logging contain sensitive data?
No
- Does this PR include or remove any sensitive information itself?
No

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
